test/tests/serial/mail_with_imap.bats

```diff
@@ -38,7 +38,7 @@ teardown_file() {
 }
 
 @test "checking saslauthd: rimap smtp authentication" {
-  run docker exec mail_with_imap /bin/sh -c "nc -w 5 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/smtp-auth-login.txt | grep 'Authentication successful'"
+  run docker exec mail_with_imap /bin/sh -c "nc -w 5 0.0.0.0 25 < /tmp/docker-mailserver-test/auth/smtp-auth-login.txt | grep 'Error: authentication not enabled'"
```

Comment:
I agree with @polarathene; I'd like to see a test for authenticating on a different port as well :)

Reply:
Yes, I'm agree with it. Added